ask-sdk-local-debug/src/com/amazon/ask/localdebug/lambdaContext/DebugLambdaContext.java

```diff
@@ -60,6 +60,16 @@ public String getFunctionName() {
     /**
      * Defaults to returning null cognito identity.
      */
+    @Override
+    public String getFunctionVersion() {
+        return null;
+    }
+
+    @Override
```

Comment:
Add a doc string here for fixing travis issues

Reply:
updated